packages/angular/projects/ui-angular/src/lib/components/authenticator/components/federated-sign-in/federated-sign-in.component.ts

```diff
@@ -41,18 +43,10 @@ export class FederatedSignInComponent implements OnInit {
 
   private setFederatedTexts() {
     const { route } = this.authenticator;
-    const federatedText = route === 'signUp' ? 'Up' : 'In';
-    this.signInAmazonText = translate<string>(
-      `Sign ${federatedText} with Amazon`
-    );
-    this.signInAppleText = translate<string>(
-      `Sign ${federatedText} with Apple`
-    );
-    this.signInFacebookText = translate<string>(
-      `Sign ${federatedText} with Facebook`
-    );
-    this.signInGoogleText = translate<string>(
-      `Sign ${federatedText} with Google`
-    );
+
+    this.signInAmazonText = getSignInWithFederationText(route, 'amazon');
+    this.signInAppleText = getSignInWithFederationText(route, 'amazon');
```

Comment:
Ooh typo, should be `apple` 🍎

Reply:
Thank you, this typo transferred over to snapshots too [🤦](https://emojipedia.org/person-facepalming/#:~:text=%F0%9F%A4%A6%20Person%20Facepalming&text=Used%20to%20display%20frustration%20or,to%20the%20Picard%20Facepalm%20meme.)